javascript/ql/src/semmle/javascript/frameworks/HTTP.qll

```diff
@@ -240,14 +240,22 @@ module HTTP {
    */
   private predicate isDecoratedCall(DataFlow::CallNode call, DataFlow::FunctionNode decoratee) {
     // indirect route-handler `result` is given to function `outer`, which returns function `inner` which calls the function `pred`.
-    exists(int i, Function outer, Function inner |
+    exists(int i, DataFlow::FunctionNode outer, HTTP::RouteHandlerCandidate inner |
```

Comment:
I should have called this out earlier, but can we perhaps include a source code example that relates `decoratee`, `outer` and `inner`?